Dockerfile

```diff
@@ -16,7 +16,7 @@ FROM gostable AS go-linux
 
 FROM go-${TARGETOS} AS build-base-bullseye
 
-# Copy the build utiltiies
+# Copy the build utilities
```

Comment:
Think it would be better if we would bump the `LIBGIT2_TAG` in this file as well (to `libgit2-1.1.1-2`).